en/docs/tutorials/integrating-services-tutorial.md

```diff
@@ -0,0 +1,810 @@
+# Integrating Services
+
+In this tutorial, you can learn how to integrate multiple services via Choreo. Here, let's consider a simple example where an online shoe store uses an application that captures online orders. For each order, the application also needs to generate a notification for the sales manager as well as update the inventory recordsinventory records and notify the status to the client who places the order.
+
+## Prerequisites
+
+The following are required to try out this tutorial:
+
+- A Google account. In this Google account, open the Google Drive and create a Google sheet named `inventory` with the following table:
+
+    ![Google Sheet Extract](../assets/img/tutorials/inventory-service-google-sheet-extract.png){.cInlineImage-half}
+    
+    | **item**    | **count** |
+    |-------------|-----------|
+    | trainers    | 100       |
+    | loafers     | 80        |
+    | kittenheels | 60        |
+    | maryjanes   | 40        |
+                            
+- A Twilio account.
+
+## Step 1: Create a service to manage inventory records
+
+In this step, you are creating a service that does the following:
+
+- Reading order requests captured by the `orders` service that you will create in step 3.
```

Comment:
These should be explained above. When I come to create the service, I would want to know how to create the service asap.